RecoTracker/LSTCore/interface/EndcapGeometryDevSoA.h

```diff
@@ -4,6 +4,8 @@
 #include "DataFormats/SoATemplate/interface/SoALayout.h"
 #include "DataFormats/Portable/interface/PortableCollection.h"
 
+#include "RecoTracker/LSTCore/interface/Common.h"
```

Comment:
Why is this needed? Just for clarity?

Same for the rest of the files.

Reply:
In this particular case it's not needed, but in some of them we need it to use array types defined in that header. I ended up just adding it to all of them, but I can remove the unneeded ones

Reply:
No need, let's leave it like this. Thanks!

Reply:
Also, for the DeviceCollection ones it's needed so that it knows that `ALPAKA_ACCELERATOR_NAMESPACE::lst` reuses everything in `lst`.